zulipterminal/api_types.py

```diff
@@ -34,7 +34,9 @@ class Message(TypedDict, total=False):
     content: str
     timestamp: int
     client: str
-    subject: str  # Only for stream msgs.
+    subject: str
+    # Changing subject -> topic as per the migration in Zulip 2.0
+    topic: str  # Only for stream msgs.
```

Comment:
I'd suggest we comment out the subject, with a trailing comment referring to the migration.
In an ideal world that would mean using subject would cause a type error, but currently this TypedDict has total=False, so that won't happen (yet).